src/service/templates/service.filter.js

```diff
@@ -1,13 +1,17 @@
-module.exports = function() {
-  if(process.env.NODE_ENV !== 'production') {
-    console.warn(`You are using the default filter for the \`{{options.name}}\` service (in ${__dirname}).`);
-    console.warn(`This means all clients will get every real-time event.`);
-    console.warn(`For more information how to filter events see http://docs.feathersjs.com/real-time/filtering.html`);
-    console.warn(``);
+module.exports = function () {
+  function log () {
+    if (process.env.NODE_ENV !== 'testing') {
+      console.log(...arguments);
+    }
   }
 
-  return function() {
-    console.log(`{{options.name}} filter returned: true`);
+  log(`You are using the default filter for the \`{{options.name}}\` service (in ${__dirname}).`);
+  log(`This means all clients will get every real-time event.`);
+  log(`For more information how to filter events see http://docs.feathersjs.com/real-time/filtering.html`);
+  log(``);
+
+  return function () {
+    log(`{{options.name}} filter returned: true`);
```

Comment:
@slajax ya it's a tradeoff either way. For sure people have apps in the wild that are leaking info right now so I'd almost prefer to go more secure first in this instance. Thoughts @feathersjs/core-team?